syntax="proto3";

package ErzeugerRpc;

service ErzeugerStatus {

  rpc SetStatus(KomponentenID) returns (Empty) {}

}

message Empty {

}

message KomponentenID {
  int32 id = 1;
}